winui/AvatarView/content-types.md

```diff
@@ -1,10 +1,267 @@
 ---
 layout: post 
```

Comment:
@MadhumithanJayapathi Change the image names as we discussed.

Reply:
@Manivannan-E  Changed the image names as we discussed.